cmd/kubelet/app/server.go

```diff
@@ -166,7 +167,7 @@ func NewKubeletServer() *KubeletServer {
 		HealthzPort:                 10248,
 		HealthzBindAddress:          util.IP(net.ParseIP("127.0.0.1")),
 		RegisterNode:                true, // will be ignored if no apiserver is configured
-		OOMScoreAdj:                 -900,
+		OOMScoreAdj:                 -1000,
```

Comment:
Hmmm that's a good point. What does the kernel do in case of ties? My concern is that we do want the Kubelet to be killed when it is missbehaving and -1000 is a special "unkillable" value.